sql/catalyst/src/main/scala/org/apache/spark/sql/types/Decimal.scala

```diff
@@ -240,9 +240,11 @@ final class Decimal extends Ordered[Decimal] with Serializable {
 
   def toFloat: Float = toBigDecimal.floatValue
 
+  private def rawLongValue: Long = longVal / POW_10(_scale)
```

Comment:
let's still use the old name `actualLongVal`

Reply:
OK